include/hashinator/hashinator.h

```diff
@@ -148,7 +180,7 @@ class Hashmap {
 #endif
    };
 
-   Hashmap& operator=(const Hashmap<KEY_TYPE, VAL_TYPE>& other) {
+   Hashmap& operator=(const Hashmap<KEY_TYPE, VAL_TYPE,Allocator>& other) {
```

Comment:
I'd add a comment that the allocator won't be replaced (which of course makes sense)